app.sass

```diff
@@ -34,7 +35,6 @@ html, body
   width: 100vw
   margin: 0
   padding: 0
-  overflow: hidden
```

Comment:
This is sort of unrelated but had to do this as it stops the home page from scrolling. It was also recently raised in Cash's internal chat.